src/components/ModalBottomSheet/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styles from './ModalBottomSheet.module.scss';
+
+import classNames from 'classnames/bind';
+import Overlay from 'components/Overlay';
+import { useEffect } from 'react';
+import { useSwipeable } from 'react-swipeable';
+
+interface ModalBottomSheetProps {
+  isOpened: boolean;
+  children: React.ReactNode;
+  onClose: () => void;
+}
+
+const ModalBottomSheet = ({ isOpened, children, onClose }: ModalBottomSheetProps) => {
+  const cx = classNames.bind(styles);
+
+  const swipeableHandlers = useSwipeable({
+    onSwipedDown: onClose,
+  });
+
+  useEffect(() => {
+    if (isOpened) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'visible';
+    }
+  }, [isOpened]);
+
+  return (
+    <Overlay align="bottom" isOpened={isOpened} onClick={onClose}>
+      <div
+        className={cx('modal-container', {
+          [cx('modal-container--visible')]: isOpened,
+          [cx('modal-container--hidden')]: !isOpened,
```

Comment:
[classnames](https://github.com/JedWatson/classnames#alternate-bind-version-for-css-modules) 사용법에 따르면 한 번 styles 객체를 바인딩한 이후에는 cx로 다시 한번 래핑하지 않아도 될 것 같습니다!

```suggestion
          'modal-container--visible': isOpened,
          'modal-container--hidden': !isOpened,
```